includes/wikia/api/DWDimensionApiControllerSQL.class.php

```diff
@@ -16,10 +16,15 @@ class DWDimensionApiControllerSQL {
            city.city_vertical      AS vertical_id,
            IFNULL(city.city_cluster, \'c1\') AS cluster,
            city.city_created       AS created_at,
-           0                       AS deleted
+           0                       AS deleted,
+           IF(cv_test_wiki.cv_value=\'b:1;\', \'1\', \'0\') AS is_test_wiki
```

Comment:
Is there a reason you return 1 and 0 as string instead of number?

Reply:
You're right, there is no need to keep here string value